src/protocolgame.cpp

```diff
@@ -2388,18 +2398,17 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 	}
 
 	msg.addByte(0x00); // can change pvp framing option
-	msg.addByte(0x00); // expert mode button enabled
+	msg.addByte(g_game.getWorldType() != WORLD_TYPE_PVP_ENFORCED ? 0x01 : 0x00);
```

Comment:
Since the no one has spent time to actually write new new war system, there is no reason for us to even use the pvp enfo check.
Just send 0x00 and be happy with it xD
The buttons won't do anything anyways, since there is no parser function for them (kinda~).